package.json

```diff
@@ -19,7 +19,7 @@
   "homepage": "https://github.com/microsoft/pipelines-appservice-lib#readme",
   "devDependencies": {
     "@types/jest": "^24.0.15",
-    "@types/node": "^12.6.8",
+    "@types/node": "^12.20.55",
```

Comment:
why we bumped it up?

Reply:
To fix build errors. Can you please try it locally once